src/pages/panel/components/layout.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { html, define } from '/hybrids.js';
+import { t } from '/vendor/@whotracksme/ui/src/i18n.js';
+
+export default define({
+  tag: 'gh-panel-layout',
+  render: () => html`
+    <slot name="header"></slot>
+    <section id="content"><slot></slot></section>
+    <footer>
+      <a href="https://www.ghostery.com/support" target="_blank">
+        ${t('panel_menu_report_broken_site')}
+      </a>
+      <a
+        href="https://www.ghostery.com/submit-a-tracker"
+        target="_blank"
+        class="subscribe"
+        >${t('panel_menu_submit_tracker')}</a
+      >
+    </footer>
+  `.css`
+    :host {
+      display: flex;
+      flex-flow: column nowrap;
+      min-height: 100%;
+      width: 100%;
+      min-width: 375px;
+    }
+
+    @media only screen and (min-height: 346px) {
```

Comment:
Here is the hack - on chrome we cannot just pass `height: 100%` as it shrinks the view to less then 100px height, but as the height is related to the content, it only applies when Chrome already calculates the height, so in the result using media does not break it. And what most important - this rule is required on macOS Safari to limit content height and display scrollbar correctly.

Reply:
sweet!